czicompress/pack/libczicompressc.nuspec

```diff
@@ -2,15 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     <id>libczicompressc</id>
-    <version>0.5.2</version>
+    <version>$token_version$</version>
     <title>Native runtimes for CZI ZStd compression</title>
     <authors>Carl Zeiss Microscopy</authors>
-    <owners>Carl Zeiss Microscopy</owners>
     <description>Native runtimes for CZI ZStd compression</description>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <license type="expression">MIT</license>
-    <projectUrl>https://github.com/zeissmicroscopy/czicompress/</projectUrl>
     <copyright>© Carl Zeiss Microscopy GmbH and others. All rights reserved.</copyright>
-    <repository type="git" url="https://github.com/ZEISS/czicompress" branch="main" commit="40a2ba8489544626c2481ca95177b3235b5c0de7" />
+    <repository type="git" url="https://github.com/ZEISS/czicompress" commit="$token_commit$"/>
```

Comment:
Do it like https://www.fuget.org/packages/Microsoft.CodeAnalysis.PublicApiAnalyzers/3.3.2, i.e. only specifying commit (and not branch) - mainly to avoid issues with "hierarchical" branch names and full refs (like origin/refs/heads/<branch name>). On top, branches are only pointers - commits are enough.

Background: https://github.com/NuGet/NuGetGallery/issues/7537